UK/LogonDetails.py

```diff
@@ -44,6 +57,58 @@
 CPDLC = input("Enter Hoppie's Code: ")
 os.system('cls')
 
+# VCCS PTT Key
+
+print("Push a key you want to use for VCCS. (Note: Keyboard only)")
+PTT = str(int(hex(keyboard.read_event().scan_code), 16) << 16)
```

Comment:
The keybind prompt character may be transferred to the next prompt as input.

Reply:
Yea not too sure on how to fix, all the methods I've tried still result in the input being transferred to the next prompt.

Reply:
I think I've resolved this somehow somewhere else. I'll get back to you on it

Reply:
Could this be the result of holding the key down for too long? You should make sure to check for the key-up event when reading keyboard input